### ISSUE TITLE
Container startup fails when using `codecentric/keycloakx` Helm chart

### ISSUE THREAD

**Issue by joostfarla** [p2-inc/phasetwo-containers#53]
When using the container image with the [codecentric/keycloakx Helm chart](https://artifacthub.io/packages/helm/codecentric/keycloakx), the container will not get into a healthy state. The reason is that the management server is listing on the `https` scheme, while the startup/readiness/health/ probes will request the `http` scheme. 

Pod logs:

```
  Warning  Unhealthy  21s (x3 over 31s)  kubelet            Startup probe failed: Get "http://10.1.1.17:9000/health": dial tcp 10.1.1.17:9000: connect: connection refused
```

Keycloak logs:

```
2024-07-14 09:41:17,242 INFO  [io.quarkus] (main) Keycloak 25.0.1 on JVM (powered by Quarkus 3.8.5) started in 15.547s. Listening on: http://0.0.0.0:8080 and https://0.0.0.0:8443. Management interface listening on https://0.0.0.0:9000.
```

For now, I have worked around the issue by overriding all probes in the `values.yaml` file, adding the `https` scheme.
Is this the right way to resolve this issue, or could TLS be fully disabled, since I'm running Keycloak behind a reverse proxy (Traefik)? 

In the default `quay.io/keycloak/keycloak` image, TLS seems to be fully disabled, as can bee seen in the logs:
```
2024-07-13 11:24:55,984 INFO  [io.quarkus] (main) Keycloak 25.0.0 on JVM (powered by Quarkus 3.8.5) started in 5.745s. Listening on: http://0.0.0.0:8080. Management interface listening on http://0.0.0.0:9000.
```

Thanks in advance!

**Comment by xgp** [comment]
Does this work properly with the standard Keycloak 25.0.1 image? Seems like an issue with the chart you’re using rather than the image.

**Comment by joostfarla** [comment]
Yes, in the default image (`quay.io/keycloak/keycloak`) TLS is disabled for the default server. This results in the management server listening on the `http` schema, making the probes succeed. I have now also tested with the `keycloak/keycloak` image, and this also succeeds (TLS is disabled).

I tried figuring out what makes TLS being enabled for the `quay.io/phasetwo/phasetwo-keycloak` image (by comparing the env vars in the running container), but I did not find any differences.

**Comment by xgp** [comment]
I wonder if it's related to just having a certificate:

https://github.com/p2-inc/phasetwo-containers/blob/main/Dockerfile#L38

We originally did this when the Quarkus version (~17) came out because it would complain if it wasn't there. Can you build without that line and see if that solves the problem?

**Comment by joostfarla** [comment]
Yes, that does indeed solve the problem. TLS is now disabled and probes succeed:

```
2024-07-15 11:39:08,991 INFO  [io.quarkus] (main) Keycloak 25.0.1 on JVM (powered by Quarkus 3.8.5) started in 14.444s. Listening on: http://0.0.0.0:8080. Management interface listening on http://0.0.0.0:9000.
```

**Comment by xgp** [comment]
Fixed in 557e4be7885fb6a6aad7736ade9f2ae7c59c315f
